src/inmanta/protocol/methods.py

```diff
@@ -59,7 +59,7 @@ async def convert_resource_version_id(rvid: model.ResourceVersionIdStr, metadata
         raise exceptions.BadRequest(f"Invalid resource version id: {rvid}")
 
 
-ENV_OPTS: Mapping[str, ArgOption] = {
+ENV_OPTS: Dict[str, ArgOption] = {
```

Comment:
It made mypy unhappy at other places, I started changing it at those places but each time I changed it at one place I needed to fix it somewhere else so I gave up as I don't think it is that important. If you insist I will try to fix it everywhere :p

Reply:
No, this one is fine. It surprises me but it's not really important. The ones about `object` are because the argument doesn't fly there: if the change makes mypy complain, it means there is an issue you were hiding with the `Any`.